db/models/album.js

```diff
@@ -1,13 +1,20 @@
 module.exports = (sequelize, DataTypes) => {
   const Album = sequelize.define('album', {
-    id: DataTypes.INTEGER,
+    id: {
```

Comment:
You're code looks really clean, so I guess I'll offer just stylistic preference that doesn't really need to be included, but just a thought. In your db/models you don't need to specify ID since your db/migrations already has the id defined.